### PR TITLE
Adding "stopwordsiso"

### PR DESCRIPTION
**Pull request by dcfidalgo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dcfidalgo** [comment]
I confirm that I'm willing to be listed as a maintainer.

**Comment by dcfidalgo** [comment]
@conda-forge-admin, please ping team

This should be ready to review, thank you and have a nice day!

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:13]:
```suggestion
  number: 0
  noarch: python
```
It seems like this package can be noarch but please double-check this.

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:22]: Are you sure `setuptools` is required?

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:26]:
```suggestion
test:
  imports:
    - stopwordsiso
  commands:
    - pip check
  requires:
    - pip
```

This just double checks that the dependencies are correct.

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:29]:
```suggestion
  home: https://github.com/stopwords-iso/stopwords-iso
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:33]:
```suggestion
  summary: Collection of stopwords for multiple languages. Using ISO 639-1 language code.
```

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:35]:
```suggestion
```

If they're not being used, you can remove them.

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/stopwordsiso/meta.yaml
Comment (dcfidalgo) [reply recipes/stopwordsiso/meta.yaml:23]: Yes, since it uses `import pkg_resources` to find the packaged "stopwords-iso.json". I checked, and the build failed without this dependency.

**Comment by dcfidalgo** [comment]
@BastianZim Thanks a lot for the review, I committed most of your suggestions.

File: recipes/stopwordsiso/meta.yaml
Comment (BastianZim) [inline recipes/stopwordsiso/meta.yaml:32]: The checks now seem to be here twice. Can you please clean up this part, then this should be ready to go!

Comment (dcfidalgo) [reply recipes/stopwordsiso/meta.yaml]: arg, sorry, not sure what happened here. fixed it.

Comment (BastianZim) [reply recipes/stopwordsiso/meta.yaml]: Thanks! Maybe a GitHub error. Looks good - will merge this when the tests pass. 

Thanks for your contribution!

**Review by BastianZim — APPROVED** [review]

**Pull request merged**
